test/compatibility/compatibility_test_for_releases.sh

```diff
@@ -112,10 +112,15 @@ build_branch()
         # Old releases didn't have these enabled, need to make it consistent.
         config+="-DENABLE_LZ4=0 -DENABLE_ZLIB=0 -DENABLE_ZSTD=0 "
         config+="-DWT_STANDALONE_BUILD=0 "
-        # Use the stable MongoDB toolchain for this build.
-        config+="-DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_stable_gcc.cmake "
         # Disable cppsuite - not all versions build with the toolchain
-        config+="-DENABLE_CPPSUITE=0"
+        config+="-DENABLE_CPPSUITE=0 "
+
+        # Use the stable MongoDB toolchain if it exists
+        toolchain="$PWD/cmake/toolchains/mongodbtoolchain_stable_gcc.cmake"
```

Comment:
Where/When does this file not exist? I thought this file had always been part of our source code.

Reply:
Here is the [history](https://github.com/wiredtiger/wiredtiger/commits/develop/cmake/toolchains/mongodbtoolchain_stable_gcc.cmake) - why does this issue appear now?

Reply:
https://github.com/wiredtiger/wiredtiger/tree/mongodb-6.0.2/cmake/toolchains

Reply:
It's just hard for me to believe that we never had the combination with `mongodb-6.0` before this time. If we always pick randomly between the few branches we have, we should have hit this in the past? What do you think @lukech?

Reply:
@ivankochin and I had a chat offline - it makes sense now. We used to have the v4 path hardcoded and we would not use the "stable" alias until a change that came in a week ago.